components/uid.rst

```diff
@@ -0,0 +1,155 @@
+.. index::
+   single: UID
+   single: Components; UID
+
+The UID Component
+=================
+
+    The UID component provides utilities to work with `unique identifiers`_ (UIDs)
+    such as UUIDs and ULIDs.
+
+.. versionadded:: 5.1
+
+    The UID component was introduced in Symfony 5.1 as an
+    :doc:`experimental feature </contributing/code/experimental>`.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/uid
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+This component provides different classes to generate each type of UID. The
+following sections explain each of them.
```

Comment:
```suggestion
This component provides different classes to generate different unique
identifiers. The following sections explain each of them.
```

UID is "just" a name (like "functions"), not a specific specification. So I think "each type of UID" does really say something.